### DISCUSSION TITLE
Subscribe page modifications

### DISCUSSION THREAD

**Discussion by greg-konv** [requarks/wiki#6420] [Help / Questions] [answered]
Hello,

is it possible to allow users to subscribe a page to receive a notification when a page is modified? I saw few feedbacks marked as in plan but I didn't found any further information about it:

- https://feedback.js.wiki/wiki/p/notifications-for-page-updates
- https://requarks.canny.io/wiki/p/watch-page

In my case I have an installation with +100 readers, +20 editors and some hundreds documents, users are starting to ask to get notifications when something is updated.

**Comment by NGPixel** [comment, marked as answer]
This is planned for v3.
